File: EducationService/src/main/java/com/stackroute/educationservice/domain/Chicklets.java
package com.stackroute.educationservice.domain;

import lombok.AllArgsConstructor;
import lombok.Builder;
import lombok.Data;
import lombok.NoArgsConstructor;
import org.springframework.data.annotation.Id;
import javax.validation.constraints.NotNull;

@Data
@NoArgsConstructor
@AllArgsConstructor
@Builder
public class Chicklets {

    private Qualification qualification;
    private Institution institution;
    private String summary;
}

File: angular/src/app/cards/classes/chicklets.ts
import { Qualification } from "./qualification";
import { Institution } from "./institution";

export class Chicklets {
    constructor(private qualification: Qualification,
        private institution: Institution,
        private summary: string) {
        this.qualification = qualification;
        this.institution = institution;
        this.summary = summary;
    }
}